test/src/unit-sparse-global-order-reader.cc

```diff
@@ -254,7 +256,11 @@ int32_t CSparseGlobalOrderFx::read(
   if (set_subarray) {
     // Set subarray.
     int subarray[] = {1, 10};
```

Comment:
With the new subarray parameter suggested above, there will only be one call to tiledb_query_set_subarray.

Reply:
ok